src/contracts/core/AllocationManager.sol

```diff
@@ -4,21 +4,24 @@ pragma solidity ^0.8.27;
 import "@openzeppelin-upgrades/contracts/proxy/utils/Initializable.sol";
 import "@openzeppelin-upgrades/contracts/security/ReentrancyGuardUpgradeable.sol";
 import "../mixins/Deprecated_OwnableUpgradeable.sol";
+import "../mixins/SplitContractMixin.sol";
 import "../mixins/PermissionControllerMixin.sol";
 import "../mixins/SemVerMixin.sol";
 import "../permissions/Pausable.sol";
 import "../libraries/SlashingLib.sol";
 import "../libraries/OperatorSetLib.sol";
-import "./AllocationManagerStorage.sol";
+import "./storage/AllocationManagerStorage.sol";
 
 contract AllocationManager is
     Initializable,
     Deprecated_OwnableUpgradeable,
     Pausable,
     AllocationManagerStorage,
     ReentrancyGuardUpgradeable,
+    SplitContractMixin,
```

Comment:
Thoughts on adding this to the end to not mess with the inheritance chain? I know it has no storage slots - but still...